applications/Chat/coati/dataset/sft_dataset.py

```diff
@@ -1,4 +1,4 @@
-#    Copyright 2023 Rohan Taori, Ishaan Gulrajani, Tianyi Zhang, Yann Dubois, Xuechen Li
+#    Copyright 2023 Rohan Taori, Ishaan Gulrajani, Tianyi Zhang, Yann Dubois, Xuechen Li, Pengpeng Zhang
```

Comment:
This copyright means the code is developed by Rohans and we cite their work here.